homeassistant/components/fully_kiosk/services.py

```diff
@@ -62,6 +65,24 @@ async def async_start_app(call: ServiceCall) -> None:
         for coordinator in await collect_coordinators(call.data[ATTR_DEVICE_ID]):
             await coordinator.fully.startApplication(call.data[ATTR_APPLICATION])
 
+    async def async_set_config(call: ServiceCall) -> None:
+        """Set a Fully Kiosk Browser config value on the device."""
+        for coordinator in await collect_coordinators(call.data[ATTR_DEVICE_ID]):
+            # Fully API has different methods for setting string and bool values.
+            # check if call.data[ATTR_VALUE] is a bool
+            if (
+                isinstance(call.data[ATTR_VALUE], bool)
+                or call.data[ATTR_VALUE].lower() == "true"
+                or call.data[ATTR_VALUE].lower() == "false"
+            ):
+                await coordinator.fully.setConfigurationBool(
```

Comment:
Will this automatically convert booleans to `true` / `false` while also accepting those as strings?

Reply:
Python lib should accept both